checks/caesar.py

```diff
@@ -0,0 +1,47 @@
+import os
+import sys
+
+sys.path.append(os.getcwd())
+from check50 import File, Test, Error, check
+
+class Vigenere(Test):
```

Comment:
Sure, @brianyu28, for now, why don't we just mimic `unittest`'s own class names, since we're just tweaking them for our purposes. Seems easier for new readers (myself included) not to introduce new semantics.